pkg/core/interop/crypto/interop.go

```diff
@@ -21,5 +23,17 @@ func init() {
 
 // Register adds crypto interops to ic.
 func Register(ic *interop.Context) {
-	ic.Functions = append(ic.Functions, cryptoInterops)
+	for _, f := range cryptoInterops {
```

Comment:
Because it's not performance-critical and you can just append/sort it. And frankly I think in these tests you might as well do `ic.Functions = cryptoInterops`, they shouldn't be using other functions.

Reply:
As I remember it's initial purpose was to be used during consensus payloads verification, which is no longer needed.
The changes are not needed, we can even remove the function and inline in tests, there are only 2 of them.

Reply:
It makes things simpler, fixed.